packages/terra-form-select/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Unreleased
 * Fixed
   * Fixed `Native Select` to allow updating the value prop in subsequent renders.
+  * Removed index to avoid screen reader reading index twice in `terra form single select`.
```

Comment:
```suggestion
  * Fixed screen reader reading the index twice in `terra form single select`.
```

Reply:
this change will affect only single select because values for `totalOptions` and `index` is set only on [clone()](https://github.com/cerner/terra-core/blob/a066d1f08dc384e15f4351631e468f4b96ab12ac/packages/terra-form-select/src/single/Menu.jsx#L395-L396) function of single-select/menu.jsx 
remaining variants seems to be not using `totalCount` and `index` due to which below code block was skipped for other variants.
```
  if (SharedUtil.isMac() && index && totalOptions) {
    label = `${display} (${index} ${ofText} ${totalOptions})`;
  }
```

Reply:
That makes sense. Thanks!